### PR TITLE
Move testing to github actions

### PR DESCRIPTION
**Pull request by mhvk**
Since travis is disappearing, we need to move our tests. Who knows, this might even work!

### CODE REVIEW

**Comment by mhvk** [comment]
@astrofrog - if you have a moment, would you be able to check why actions are not running on this PR?  I forget whether one has to set something for it to happen...

File: .github/workflows/ci_workflows.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+          - name: MacOS (with clang)
+            os: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: bootstrap
+      run: bootstrap.sh
```
Comment (avalentino) [inline .github/workflows/ci_workflows.yml:29]: Probably this should be
```
./bootstrap.sh
```
See also avalentino/erfa@03717dfe84d81fc036cfe4d106ab4b55949e6fd9

File: .github/workflows/ci_workflows.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - name: configure
+      run: configure --disable-shared
```
Comment (avalentino) [inline .github/workflows/ci_workflows.yml:31]: Same here

**Review by avalentino — CHANGES_REQUESTED** [review]

**Comment by avalentino** [comment]
On MacOSX we need to install "aclocal"?
See test failure in https://github.com/avalentino/erfa/actions

**Comment by mhvk** [comment]
I had the `./...sh` versions before, but it didn't seem anything even ran. I've now reverted to it (and set `fail-fast` to `false` so one can check independently). Not sure why nothing is running automatically here!

**Comment by mhvk** [comment]
@avalentino - if you have a working configuration (even with `clang` on linux), obviously fine to make that into a PR and merge it instead! I vaguely recall one needs to merge something first to get going...

**Comment by avalentino** [comment]
I have just temporarily enabled the current branch avalentino/erfa@6f1a8af3f928a7d77a090f34d4b8f3c46f8b978e

**Comment by avalentino** [comment]
OK @mhvk if you cherry-pick this commit avalentino/erfa@f576efa8d5edb5b3999467093205b279270df08a it should work also on MacOSX

**Comment by mhvk** [comment]
OK, thanks! Cherry-picked that one. It still doesn't run automatically, do you know if we just need to merge first, or have to explicitly enable it?

**Comment by astrofrog** [comment]
It will only run on PRs once merged (but you can check how it runs currently on your fork)

**Comment by avalentino** [comment]
I have checked it in my fork:  https://github.com/avalentino/erfa/actions.
IMHO it is safe to merge.

**Comment by mhvk** [comment]
OK, since it is checked, let's merge!

**Comment by mhvk** [comment]
Thanks, both!

**Pull request merged**